scripts/testnet/sovereignBridge/config/token.snippets.sh

```diff
@@ -47,17 +53,22 @@ depositTokenInSC() {
            ${AMOUNT_TO_TRANSFER} \
            str:deposit \
            ${WALLET_ADDRESS} \
+        --outfile=${OUTFILE} \
         --recall-nonce \
         --wait-result \
         --send || return
+
+    printTxStatus ${OUTFILE}
 }
 
 issueTokenSovereign() {
     manualUpdateConfigFile #update config file
 
-    local TOKENS_TO_MINT=$(echo "scale=0; $INITIAL_SUPPLY_SOVEREIGN*10^$NR_DECIMALS_SOVEREIGN/1" | bc)
+    echo "Issuing fungible token on sovereign chain..."
```

Comment:
Can't we use a base func to issue token as you did before?
This code seems duplicated with the one from normal chain

Reply:
It's just an mxpy command with arguments from config. Will live it as it is for now. Maybe in the future we'll even delete these issue examples..